### DISCUSSION TITLE
@react/spectrum -> @adobe/react-spectrum missing components?

### DISCUSSION THREAD

**Discussion by ianjwhite99** [adobe/react-spectrum#3574] [Q&A] [answered]
Hi,

I was going through `packages/@react-spectrum` and noticed that there are a few components that aren't included in the `packages/@adobe/react-spectrum` bundle available on NPM. 

For example there is a toast component and alert component that don't seem to be included with the main @adobe/react-spectrum package. 

Is this because those components are unreleased? I noticed in the `package.json` the name included `X.X.X-alpha.1` but I couldn't find anything discussing their continued development or future release.

**Comment by snowystinger** [comment, marked as answer]
We don't add components to @adobe/react-spectrum until they've had their official 3.0.0 release. alphas, betas, rc's are all excluded. Those components in particular we started development on, but then they were deprioritized in favor of other components.

**Reply by ianjwhite99** [reply]
Makes sense! Thanks for the quick response 🙂
